sdk/core/azure-core/src/main/java/com/azure/android/core/implementation/util/serializer/DateTimeRfc1123Serializer.java

```diff
@@ -31,10 +30,6 @@ public static SimpleModule getModule() {
     @Override
     public void serialize(DateTimeRfc1123 value, JsonGenerator jgen,
                           SerializerProvider provider) throws IOException {
-        if (provider.isEnabled(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)) {
-            jgen.writeNumber(value.dateTime().getTime());
-        } else {
-            jgen.writeString(value.toString()); //Use the default toString as it is RFC1123.
-        }
+        jgen.writeString(value.toString()); //Use the default toString as it is RFC1123.
```

Comment:
`SerializationFeature.WRITE_DATES_AS_TIMESTAMPS` feature is disabled for Azure use-cases hence removing check for it.